propagation/ogc_processes.md

```diff
@@ -1,33 +1,25 @@
-# OGC API Processes: file-based processing for existing software
+# OGC Application Package - file-based processing for existing software
 
-The OGC API processes specification allows you to expose basically any type of (processing) web service. As a service
-designer, you get the full freedom and responsibility in the definition of your service.
+Another option to create an APEx-compliant service is to package your software using Docker containers. This approach is very generic and is particularly relevant if you have an existing piece of complex software that you want to make available as a service.
```

Comment:
Please clarify better web service vs Docker container.
The end result that we need according to SoW is an on-demand web service, so based on OGC API Processes.
OGC Application package deployed on hosting platform is the recommended approach to reach that goal. Avoiding confusion is key here!

Then there's also the question: are projects or hosting platforms also 'allowed' to just provide the service directly, for instance without docker container?